app/views/faq/show.html.erb

```diff
@@ -0,0 +1,89 @@
+      <dd>
+        <p>Here are the latest papers using the scenicness data:</p>
+
+        <p>
+          Seresinhe, C. I., Preis, T., Mackerron, G. & Moat, H. S. (2019). Happiness is greater in more scenic locations. <cite>Scientific Reports, 9</cite>, 4498.<br>
+          <a href="https://www.nature.com/articles/s41598-019-40854-6">https://www.nature.com/articles/s41598-019-40854-6</a><br><br>
+
+          Seresinhe, C. I., Moat, H. S., & Preis, T. (2018). Quantifying scenic areas using crowdsourced data. <cite>Environment and Planning B: Urban Analytics and City Science, 45</cite>, 567-582.<br>
+          <a href="https://journals.sagepub.com/doi/10.1177/0265813516687302">https://journals.sagepub.com/doi/10.1177/0265813516687302</a><br><br>
+
+          Seresinhe, C. I., Preis, T., & Moat, H. S. (2017). Using deep learning to quantify the beauty of outdoor places. <cite>Royal Society Open Science, 4</cite>, 170170.<br>
+          <a href="https://rsos.royalsocietypublishing.org/content/4/7/170170">https://rsos.royalsocietypublishing.org/content/4/7/170170</a><br><br>
+
+          Seresinhe, C. I., Preis, T., & Moat, H. S. (2015). Quantifying the impact of scenic environments on health. <cite>Scientific Reports, 5</cite>, 16899.<br>
+          <a href="https://www.nature.com/articles/srep16899">https://www.nature.com/articles/srep16899</a><br><br>
+
+          If you would like to stay updated on the latest developments on this research then feel free to get in touch with the <a href="https://datasciencelab.co.uk/contact-us">Data Science Lab</a> or follow us at <a href="https://twitter.com/thedatascilab">@thedatascilab</a>.
+        </p>
+      </dd>
+
+      <dt>Can I download the data generated by this site?</dt>
+      <dd>
+        <p>Sure, we're happy to
+          make the data available for other people to use under the <a href="http://opendatacommons.org/licenses/odbl/">Open Database Licence</a>.
+        </p>
+
+        <p>
+          <a href="/votes.tsv">votes.tsv</a> is a <acronym title="tab separated value">TSV</acronym>
+          file of all photos that have been rated 3 times or more. The
+          columns are ID, latitude, longitude, average rating, population
+          variance, the votes (comma separated), and the Geograph URI for the
+          photo. The current file contains all the votes posted until February 2015.
+        </p>
+
+        <p>There are 217,000 images (one for each of nearly 95% of the 1km grid
+          squares in Great Britain), and we're making sure each one is rated 3 times by
+          different people, to help limit abuse, or the impact of people with
+          terrible taste.
+        </p>
+      </dd>
+
+      <dt>Who built it?</dt>
+      <dd>
+        <p>
+          This site was originally built by <a href="https://www.dxw.com/">The
+          Dextrous Web (now dxw)</a> for <a href="http://www.mysociety.org">mySociety</a>.
```

Comment:
I think we can probably just say `dxw` here instead of using the full name.

Reply:
Hah, yes, it was a copy and paste, except for where it said `"(now Dextrous Digital)"`, which I've already changed to `"(now dxw)"`. I can change it to just "built by dxw".

Reply:
That's my reckon.

Reply:
Done. I've also added the link to the votes TSV, which they've uploaded in the meantime.